docs/CloudFerry_Beginner_Demo_How-To.txt

```diff
@@ -0,0 +1,279 @@
+CloudFerry
+Cloud Resource Migration Tool
+Demo How-To
+****
+
+Introduction
+============
+
+Assumed Knowledge: You should be a Linux/Unix system administrator or
+engineer with at least a passing familiarity with programming languages
+(CF is written in Python), networking, databases, Unix text editors,
+large scale storage appliances, RESTful applications and, of course,
+substantial knowledge of and experience with OpenStack including at the
+command line.
+
+CloudFerry is a highly configurable tool and as such there are more ways
+to use it than can be described in a brief document. This demo will
+assist you in creating a pair of virtual OpenStack clouds, adding
+tenants and configuring resources for you to migrate and will then
+perform a full migration. That is only a part of what CloudFerry can do.
+
+There are some terms which users of CloudFerry need to understand in the
+context they’re being used.
+
+Migration: Moving resources and workloads from one OpenStack cloud to
+another.
+
+Evacuation: Forced moving of all resident VM’s from a particular
+physical node or set of physical nodes in order to free those nodes up
+for either maintenance or migration activities.
+
+Condensation: Creation of a schedule file which determines how to move
+VM’s around on a source cloud in order to minimize the number of
+physical nodes in use. This is the pre-work to Evacuation. Condensation
+does not change anything. It only creates a schedule.
+
+Workload: The simplest *definition* of a *workload* is a single virtual
+server with all resources that were used to create it.
+
+Resource: tenants, users (preserving their passwords or generating new
+ones), roles, flavors, images, etc…
+
+In the real world it’s almost certain that since migration implys moving
+workloads from one cloud environment to another that you’ll need to
+perform a Condensation and Evacuation run prior to migration simply to
+free up enough hardware to pull it off. Those 2 topics are not covered
+in this document. If you wish to get familiar with them, configure a
+test bed environment and use fab (condense||evacuate)… instead of fab
+migrate:configuration.ini,debug=true. Condensation will generate a
+schedule file which Evacuation will use as input. They need to be run in
+that order.
+
+
+How-To: CloudFerry for Beginners
+================================
+**Overview**
```

Comment:
`Overview` is not displayed as a header, please modify, I guess you simply need to add some `----` below.